setup.py

```diff
@@ -43,7 +43,7 @@
 install_requires = [
     'invenio-base>=1.2.2',
     'invenio-celery>=1.2.0',
-    'redis>=2.10.0',
+    'redis>=3.2.0',
```

Comment:
Is it required to bump this?

Reply:
After running tests it seems that no at the end